README-sdk.md

```diff
@@ -0,0 +1,73 @@
+<img src="website/static/operator_logo_sdk_color.svg" height="125px"></img>
+
+[![Build Status](https://travis-ci.com/operator-framework/operator-sdk.svg?branch=master)](https://travis-ci.com/operator-framework/operator-sdk)
```

Comment:
should it use the build status of upstream or downstream?

Reply:
upstream, see my new comment https://github.com/openshift/ocp-release-operator-sdk/pull/102#issuecomment-779416485